client/src/test/java/com/graphhopper/api/GraphHopperWebIT.java

```diff
@@ -24,7 +24,7 @@
 
     @Before
     public void setUp() {
-        String key = System.getProperty("graphhopper.key", "");
+        String key = System.getProperty("graphhopper.key", "369dc982-86a6-484e-95ad-669331663ca4");
```

Comment:
We should make this global to the tests as other tests then also should use this key.